scripts/build/build.js

```diff
@@ -1,7 +1,7 @@
 "use strict";
 
 const path = require("path");
-const fs = require("fs");
+const fs = require("fs").promises;
```

Comment:
Is `require('fs/promises')` prefered?

Reply:
fs/promises are only supported on Node.js 14 and above

Reply:
Doesn't matter, just alias.